src/pip/_internal/req/req_tracker.py

```diff
@@ -20,6 +20,10 @@ def __init__(self):
             self._temp_dir.create()
             self._root = os.environ['PIP_REQ_TRACKER'] = self._temp_dir.path
             logger.debug('Created requirements tracker %r', self._root)
+            # Ensure the directory is not left empty, so it does not get
```

Comment:
i propose adding the comment as content of the file in order to support debugging left behinds more easyly

Reply:
Well, it should never be left behind (you'd need a hard crash of the Python interpreter).